source/synthDrivers/sapiMobile.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: UTF-8 -*-
+#synthDrivers/sapi5.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import comtypes.client
+from comtypes import COMError
+import sapi5
+
+class SynthDriver(sapi5.SynthDriver):
+
+	name="sapiMobile"

```

Comment:
Suggestion: how about referring to this speech synthesizer as sapiOneCore, since that's how Windows does it as well? Also, it might be a more clear alternative to include OneCore in the description.